src/modules/Overlay/Module.js

```diff
@@ -47,20 +47,29 @@ module.exports = function Dynamic(options, UI, util) {
 
       function changePixel(r1, g1, b1, a1, x, y) {
 
+        var firstImagePixels = [r1, g1, b1, a1];
```

Comment:
Can we use let and const?

Reply:
Yea sure, I'll update it right away!

Reply:
@HarshKhandeparkar It's done, could ypu review it again

Reply:
Ty